packages/ai/src/ai/common/models/vision/vision.py

```
@@ -0,0 +1,413 @@
+    def postprocess(
+        model: Any,
+        raw_output: Any,
+        batch_size: int,
+        output_fields: List[str],
+        **kwargs,
+    ) -> List[Dict[str, Any]]:
+        """Postprocess vision model output."""
+        from ..extract import extract_outputs
+
+        results = []
+        for i in range(batch_size):
```

Ruff (0.15.5)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)